.github/labeler.yml

```diff
@@ -0,0 +1,3 @@
+# Always remove the safe to test label
+'safe to test':
+  - '!*'
```

Comment:
reading https://github.com/actions/labeler, I am not sure I understand our '!*'. Could you please explain why it is not '*'?

Reply:
I make use of `sync-labels: true`, so if is there a `safe to test` label with this condition we remove it

Reply:
ok, thank you